pkgs/by-name/vi/vikunja-desktop/package.nix

```diff
@@ -0,0 +1,119 @@
+{
+  lib,
+  stdenv,
+  makeWrapper,
+  makeDesktopItem,
+  callPackage,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.

Reply:
Done.